internal/pkg/service/appsproxy/proxy/server_test.go

```diff
@@ -70,13 +70,6 @@ func TestAppProxyHandler(t *testing.T) {
 	require.Equal(t, http.StatusOK, rec.Code)
 	assert.Contains(t, rec.Body.String(), "Disallow: /")
 
-	// Get style.css
-	rec = httptest.NewRecorder()
-	req = httptest.NewRequest("GET", "https://hub.keboola.local/_proxy/assets/style.css", nil)
-	handler.ServeHTTP(rec, req)
-	require.Equal(t, http.StatusOK, rec.Code)
-	assert.NotEmpty(t, rec.Body.String())
-
```

Comment:
Petr, you could have deleted it yourself when you added new assets.

Reply:
I didn't remove that, because I wanted to have some test for an asset that exists. Yes, I could use favicon.ico instead style.css